src/map/skill.cpp

```diff
@@ -12959,22 +12949,16 @@ int32 skill_castend_nodamage_id (struct block_list *src, struct block_list *bl,
 		}
 		break;
 	case RL_QD_SHOT:
```

Comment:
Why is this even in skill_castend_nodamage_id and not directly in skill_castend_damage_id?

Reply:
Good question, maybe to prevent recursive calling of the same function?